Resources/Locale/en-US/info/ban.ftl

```diff
@@ -85,3 +85,7 @@ server-ban-string-never = never
 
 # Kick on ban
 ban-kick-reason = You have been banned
+
+# banmass
+cmd-banmass-desc = Bans several players at once by nickname or ID.
+cmd-banmass-help = Usage: banmass <reason> [duration in minutes, leave out or 0 for permanent ban] <name1 name2 name3..>
```

Comment:
These lines should be moved up so they are together with the other `cmd-` locstrings in the file.
The duration doesn't seem to be optional, so remove the "leave out or" part and change `[]` to `<>`.

Reply:
oks